doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -4322,7 +4373,7 @@ DRGN-11059: Front-end module API not implemented for SoftDevice Controller
 The issues in this section are related to :ref:`nrfxlib:nrf_802154`.
 In addition to the known issues listed here, see also :ref:`802.15.4 Radio driver limitations <nrf_802154_limitations>` for permanent limitations.
 
-.. rst-class:: v2-9-0-nRF54H20-1 v2-9-1 v2-9-0 v2-8-0
+.. rst-class:: v3-0-0 v2-9-0-nRF54H20-1 v2-9-1 v2-9-0 v2-8-0
 
 KRKNWK-19574: nRF 802.15.4 Radio Driver stuck in ``RADIO RXDISABLE`` state
```

Comment:
For 802.15.4 Radio Driver: please remove the old known issue and add new known issue
```suggestion
KRKNWK-19335: nRF 802.15.4 Radio Driver incorrectly reports an ``INVALID_FCS`` error if a frame is dropped while all receive buffers are full.
The nRF 802.15.4 Radio Driver reports unjustified ``INVALID_FCS`` error in a specific scenario where at least one frame is dropped while the receive buffers limit is reached. This spurious error occurs even though the frame was not actually received with a bad FCS.

  **Affected platforms:** nRF54L Series
```